src/models/stores.ts

```diff
@@ -1,27 +1,33 @@
 import { ProblemModel, ProblemModelType } from "./problem";
 import { UIModel, UIModelType } from "./ui";
 import { UserModel, UserModelType } from "./user";
+import { DB } from "../lib/db";
```

Comment:
What does the `lib` directory signify in this arrangement?

Reply:
Library, I didn't want to put it in the models folder as it doesn't use mobx.

Reply:
Clearly "lib" => "library"; I was thinking in terms of what determines what goes in lib, to which the answer is apparently model-like things that aren't MST models. I tend to think of "lib" as the place external libraries go, which is not the usage here.